rancher/fleet
# Adapt e2e tests for local infra setup tests

* add infra setup to nightly CI
* add `infra-setup` label to tests that need our infra-setup
* don't run `infra-setup` tests where we can't load images into a local registry (AKS/GKE) and use ttl.sh instead
* dev and github scripts use `$external_ip` instead of `$url` (🚨update your dev environment if you use `$url`)

* add wrapper script for `create-zot-certs`, to keep CI scripts [isolated](https://github.com/rancher/fleet/blob/master/.github/scripts/README.md)
* update `.gitignore`
* make `dev/import-images-tests-k3d` executable
* infra binary should work without cd
* refactor gitrepo test into one describe block, some shared setup
* remove boolean parameter from one test func (:
